students/marc_charbo/lesson04/mailroom_v2.py

```diff
@@ -85,9 +92,14 @@ def run():
 
 def main():
     try:
+        logging.basicConfig(filename='myapp.log', level=logging.INFO)
```

Comment:
This should be defined at the top of your program, not within `main()`

Reply:
taking care of in v3